can_tools/__init__.py

```diff
@@ -38,13 +38,42 @@ def all_subclasses(cls):
     )
 
 
-def scrapers_for_flow(cls):
-
+def fetch_all_unblocked_scrapers(cls):
     blocked = [PennsylvaniaVaccineDemographics, TableauDashboard, CDCTestingBase]
-
     return list(
         x for x in all_subclasses(cls) if not inspect.isabstract(x) and x not in blocked
     )
 
 
-ALL_SCRAPERS: List[Type[scrapers.DatasetBase]] = scrapers_for_flow(scrapers.DatasetBase)
+ALL_SCRAPERS: List[Type[scrapers.DatasetBase]] = fetch_all_unblocked_scrapers(
+    scrapers.DatasetBase
+)
+
+
+# Data sources currently used by the CAN site/API to be included in Prefect flows.
+# Last updated 08/01/2022
+ACTIVE_SCRAPERS = [
```

Comment:
Yeah, this is a good question. Initially, liked the idea of hard-coding this because it would a) make it clear what scrapers are currently active (without having to search for `autodag` tags) and b) would reduce changes throughout the codebase. 

But, I ended up having to flag all the demographic scrapers throughout the codebase anyway, and it does come with the limitations you mention. I do like the clarity of being able to look at the list and say "here's all the active scrapers" (although even some of that is lost by the way the demographic scrapers are included), but we could also just look at the Prefect dashboard to find that, too. 

I'm more than willing to flip this logic (removing this list and actively excluding any scrapers we don't want, instead of explicitly including scrapers).

Reply:
Got it.  Yeah, I guess given where we are at now (with very little changes to this codebase and wanting to audit the set of scrapers we're running for cost reasons) I think a manually-curated list isn't a bad idea.  It would be a pain to find unneeded scrapers otherwise.  Maybe just amend the comment to say something like, "We're currently minimizing our list of active scrapers in order to reduce our Prefect costs."

Reply:
Cool, I've added that comment in feb489b546306ebcef50c86008e2fc3eab878bf3